causalml/inference/meta/tlearner.py

```diff
@@ -5,7 +5,7 @@
 from scipy.stats import norm
 from sklearn.exceptions import ConvergenceWarning
 from sklearn.neural_network import MLPRegressor
-from sklearn.utils.testing import ignore_warnings
+from sklearn.utils._testing import ignore_warnings
```

Comment:
let's use the latest version handling changes in #283

Reply:
updated with changes from #283